edenai_apis/llmengine/llm_engine.py

```
@@ -744,6 +747,16 @@ def __init__(
             **kwargs,
         )
 
+    @staticmethod
+    def map_provider(provider_name: str) -> str:
+        if provider_name is None:
+            return None
+        # Try to regex match the keys of PROVIDER_MAPPING and provider_name. The first one ot match wins
+        for key in StdLLMEngine.PROVIDER_MAPPING.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)